README.md

```diff
@@ -0,0 +1,33 @@
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+#   Portfolio mobile first
+
+> Project to helps to create your persomal portfolio
+
+
+## Built With
+
+-Major languages
+HTML,CSS
+
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job documenting your project! 📜 However it would be even great if you add a `Getting started` section describing the required steps others need to follow in order to get a local copy of your project up and running! 🛠️ 

You can follow the [Microverse preset](https://github.com/microverseinc/readme-template).